### DISCUSSION TITLE
How to export port modal fields in HFSS

### DISCUSSION THREAD

**Discussion by AleksandarToshev-eag** [ansys/pyaedt#4985] [Q&A]
Hi,

I have to analyze the port modes (the modal fields) and for the purpose I need to either export them or get some indication for the excitation modes. With the Ansys tool I can go to Port Field Display and get a picture of the port modes. However I do not find a way how to export the needed port modes distributions. And I need to do it with the Pyaedt. Can somebody help me on how to export the port modes with the Pyaedt?

**Comment by Samuelopez-ansys** [comment]
Hi @AleksandarToshev-eag , Thanks for opening this Discussion. I have created an Issue : #5021 to investigate how we could do it. 
The AEDT API does not have this capability, so we will need a workaround. I will close this discussion, and I will keep you updated in the Issue.
